src/style.css

```
@@ -0,0 +1,122 @@
+.page {
+  font-family: "Roboto", sans-serif;
+}
+
+.page__product-cards {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__photo {
+  margin: 16px 3px 0;
```

you can use `auto` here
```suggestion
  margin: 16px auto 0;
```

---

fixed